doc/spec.adoc

```diff
@@ -56,7 +56,7 @@ Each element is composed of:
 ** Mandatory `groupId` and `artifactId` elements that are the Maven coordinates of the required channel.
 ** Optional `version` to stick to a given channel version (instead of requiring the latest version of that channel). In the absence of this `version`, the latest version of the channel will be determined based on the Maven repository metadata.
 * A collection of `streams` that defines all the components installable from this channel. Each stream is composed of:
-** A required `groupId` that corresponds to Maven GroupId to pull artifacts (it is not allowed to specify `*` for the groupId).
+** A required `groupId` that corresponds to Maven GroupId to pull artifacts. Special syntax `*` can be used to match _any_ groupId. It can only be used in conjunction with `*` artifactId and only a single such stream can be defined.
```

Comment:
I do not see how this would be used. If the channel specifies one stream:
groupId=\*; artifactId=\*; versionPattern=.*\.Final

It seems to me that the installer would have to download every artifact which has a version ending in "Final" from every repository and package it into the distribution. Is that correct?

Reply:
The channel doesn't dictate what artifacts are downloaded. Installer gets a list of required artifacts from Galleon feature pack and queries the channel to see what version should be used.

So a channel with a single stream like `groupId=*; artifactId=*; versionPattern=.*.Final` would allow user to download any artifact in the underlying repository ensuring the version is always the latest one matching the pattern.